Common/Data/SubscriptionDataConfigList.cs

```diff
@@ -56,9 +56,9 @@ public SubscriptionDataConfigList(Symbol symbol)
         /// <param name="normalizationMode"></param>
         public void SetDataNormalizationMode(DataNormalizationMode normalizationMode)
         {
-            if (Symbol.SecurityType == SecurityType.Option && normalizationMode != DataNormalizationMode.Raw)
+            if ((Symbol.SecurityType == SecurityType.Option || Symbol.SecurityType == SecurityType.FutureOption) && normalizationMode != DataNormalizationMode.Raw)
```

Comment:
Not sure relevant with futures?

Reply:
I think Futures is default Raw data normalization, no errors being thrown on my end